### ISSUE TITLE
Training and validation are very different

### ISSUE THREAD

**Issue by ai-zhang-10** [Mengqi-Lei/ConDSeg#5]
Hi author, first of all thank you very much for being able to provide the code. Then I had some problems reproducing your code and would be very grateful if you could help me out. When I train my own dataset, the F1 metric can reach 89, which I think is very good, but when I test it, it is only 44, which is a very big difference, and I never know what the problem is. I also tried to test the training set, and the result is that the performance is similar to the training, also it can run up to 89, but why is it not as good on the test set. Very much looking forward for you to get back to me!

**Comment by Mengqi-Lei** [comment]
Thank you for your letter! Based on the issue you mentioned, it indicates that the model has overfitted on your dataset. This could be caused by the following factors: 1) The number of samples in the dataset is too small, making it difficult for the model to accurately learn the general pathological features, thus leading to overfitting on the training set. 2) There is a significant representational shift between the training and test sets, causing the general patterns learned on the training set to fail in the test set.

Therefore, we recommend the following attempts: 1) Try different training/test set splits, and consider adjusting the training/test set ratio. 2) Adjust the data augmentation strategy to increase the diversity of training samples.

Wishing you all the best!
